src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -113,6 +116,34 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /**
+   * Access the Conversation object with the given UUID.
+   *
+   * @return null if the UUID does not match any existing Conversation.
+   */
+  public Conversation getConversation(UUID id) {
+    for (Conversation conversation : conversations) {
+      if (conversation.getId().equals(id)) {
+        return conversation;
+      }
+    }
+    return null;
+  }
+
+  /** Access the conversations created within a day. */
+  public List<Conversation> getRecentConversations() {
+    List<Conversation> recentConversations = new ArrayList<>();
+
+    Instant aDayBefore = Instant.now().minusSeconds(86400);
```

Comment:
Define a variable

static final int SECONDS_IN_DAY = 86400;

and use that here

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names